tests/modules/test_morpheus_modules.py

```diff
@@ -245,7 +245,6 @@ def _on_next(control_msg):
 
```

Comment:
We'll need to sync on adding additional tests that use the camouflage service.  See David's upcoming PR: https://github.com/nv-morpheus/Morpheus/pull/977/files#diff-13ada51897e5cf3d7ca958ffbebce048efecf04fe47e44de8f97a045b650cce3